README.md

```diff
@@ -21,29 +21,30 @@ To quickly get started, follow the installation instructions for [Python][1] or
+| `addLayers`                 | Whether to install the Datadog Lambda library as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]).                                                                                                           |
+| `addExtension`              | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, it's required to set the `apiKey` (or `apiKMSKey`) parameter. The Datadog Lambda Extension Layer is in public preview. Learn more about the Lambda Extension Layer [here][8].                                                                                                                                         |
+| `logLevel`                  | The log level, set to `DEBUG` for extended logging.                                                                                                                                                                                                                                                                                                                                                                  |
+| `enableXrayTracing`         | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                                        |
+| `enableDDTracing`           | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                                   |
+| `enableDDLogs`              | Enable Datadog log collection for the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Does this apply when using the Datadog Forwarder?

Reply:
I think the answer to my own question is "no", that is, this setting only affects log collection if the customer is using the Lambda Extension, and not the older but still more popular Datadog Forwarder-based approach. I think we should document that here.

Reply:
I think you're correct, it doesn't apply to the Datadog Forwarder. Good call on documenting that here, will do